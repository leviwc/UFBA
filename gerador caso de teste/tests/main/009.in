28 131
21 14
20 18
3 7
13 6
19 17
4 10
23 10
23 13
2 4
6 14
7 23
1 25
27 2
17 20
3 8
6 20
14 28
12 23
21 21
21 28
6 16
15 27
26 24
13 12
1 27
26 26
1 21
11 21
8 17
11 7
16 27
27 18
10 28
9 12
14 25
2 2
16 26
18 18
11 3
21 25
11 27
14 11
13 27
18 22
28 17
9 19
13 17
11 15
10 6
24 12
27 27
4 6
13 18
3 15
18 9
2 22
19 20
10 16
25 22
18 24
9 6
16 16
7 17
11 24
23 6
2 8
21 5
26 10
4 15
23 21
10 9
4 22
22 14
17 2
13 13
24 20
8 4
5 28
1 4
16 22
10 5
26 14
19 2
11 22
11 26
7 26
11 11
14 1
10 27
18 14
4 19
16 18
9 20
28 26
8 7
9 13
3 9
16 21
7 2
15 14
5 7
28 4
12 21
28 25
19 13
1 1
11 25
2 6
2 21
1 6
5 3
20 3
21 4
28 28
20 12
1 9
14 27
6 21
12 15
22 20
12 19
3 26
13 4
25 23
16 19
8 11
5 2
22 28
7 20
4 23
19 5
