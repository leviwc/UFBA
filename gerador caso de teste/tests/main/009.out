Resultado com busca binaria: 7
3161.291000 s
Resultado com backtracking apenas: 7
0.951000 s
Resultado testando todas as possibilidades: 7
11021.431000 s
Resultado com busca binaria otimizada: 7
988.864000 s
